Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM ruby:2.4
+
+ADD . /srv/lcars
```

Comment:
I'd do the copy in two stages, like this

```Dockerfile
WORKDIR /srv/lcars

COPY Gemfile Gemfile.lock ./
RUN bundle config --global frozen 1 && bundle install
COPY . .
```

to take advantage of layer caching